bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/AbstractZkLedgerManager.java

```diff
@@ -355,7 +355,7 @@ public void processResult(int rc, String path, Object ctx, byte[] data, Stat sta
 
                 LedgerMetadata metadata;
                 try {
-                    metadata = LedgerMetadata.parseConfig(data, new ZkVersion(stat.getVersion()));
+                    metadata = LedgerMetadata.parseConfig(data, new ZkVersion(stat.getVersion()),stat.getCtime());
```

Comment:
You're not covering all error codes and stat could be null, in which case it would throw an NPE.